src/components/HeaderForm.js

```diff
@@ -713,6 +718,11 @@ class HeaderForm extends Component {
     };
   };
 
+  /* Function for simplify */
+  enableSimplify = () => {
```

Comment:
This doesn't actually "enable" simplification; it toggles it. I think toggling it is the correct behavior so the function name should be changed.